_posts/2020-04-23-putting-the-serverless-in-graphQL-with-AppSync.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title: Putting the Serverless in GraphQL with AppSync
+date: 2020-04-23
+tags: AWS AppSync Cloudformation
+author: Shermayne Lee
+---
+
+# Overview
+
+AWS AppSync and GraphQL have gained popularity in recent years. However it can be confusing and difficult to start for developers new to GraphQL concept.
```

Comment:
perhaps mention the point of this blog? An intro to Appsync.. who will find it interesting?